lms/services/canvas_api/client.py

```diff
@@ -50,6 +49,7 @@ def __init__(self, authenticated_client, request):
         """
         self._client = authenticated_client
         self._request = request
```

Comment:
I think `self._request` and the `request` argument to `__init__()` are now unused and can be removed

Reply:
True, nice :+1: